Homework3/StackPrototype/Extensions/IStackExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿public static class IStackExtensions
+{
+ // at the moment when this method is used the original stack becomes empty, is this okey ?
```

Comment:
It is fine for this task. 

In production code original stack must be unchanged, I would use cloning here.